### PR TITLE
Feature/button component

### PR DESCRIPTION
**Pull request by DavidVeVe**
Created base component structure. Added unit tests cases

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -7,9 +7,9 @@ function App() {
     <div className="App">
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
+        <h1>
```
Comment (RaulIsraelFloreslarios) [inline src/App.tsx:10]: why did you add a h1 tag instead

Comment (DavidVeVe) [reply src/App.tsx:10]: apsdoifasdklfhasd

**Review by RaulIsraelFloreslarios — APPROVED** [review]
